src/Functions/FunctionsConversion_reg.cpp

```diff
@@ -5,239 +5,4927 @@ namespace DB
+│ ABE1822D                          │ ABE1822D                 │
+└───────────────────────────────────┴──────────────────────────┘
+        )"
+    },
+    {
+        "Conversion from an integer",
+        R"(
+SELECT toIPv4(2130706433);
+        )",
+        R"(
+┌─toIPv4(2130706433)─┐
+│ 127.0.0.1          │
+└────────────────────┘
+        )"
+    }
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_toIPv4 = {20, 1};
+    FunctionDocumentation::Category category_toIPv4 = FunctionDocumentation::Category::IPAddress;
+    FunctionDocumentation documentation_toIPv4 = {description_toIPv4, syntax_toIPv4, arguments_toIPv4, returned_value_toIPv4, examples_toIPv4, introduced_in_toIPv4, category_toIPv4};
+
+    factory.registerFunction<detail::FunctionToIPv4>(documentation_toIPv4);
+
+    /// toIPv6 documentation
+    FunctionDocumentation::Description description_toIPv6 = R"(
+onverts a string or a `UInt128` form of IPv6 address to [`IPv6`](../data-types/ipv6.md) type.
+For strings, if the IPv6 address has an invalid format, returns an empty value.
+Similar to [`IPv6StringToNum`](/sql-reference/functions/ip-address-functions#IPv6StringToNum) and [`IPv6NumToString`](/sql-reference/functions/ip-address-functions#IPv6NumToString) functions, which convert IPv6 address to and from binary format (i.e. `FixedString(16)`).
+
+If the input string contains a valid IPv4 address, then the IPv6 equivalent of the IPv4 address is returned.
+)";
+    FunctionDocumentation::Syntax syntax_toIPv6 = "toIPv6(x)";
+    FunctionDocumentation::Arguments arguments_toIPv6 = {
+        {"x", "An IP address.", {"String", "UInt128"}}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_toIPv6 = {"Returns an IPv6 address.", {"IPv6"}};
+    FunctionDocumentation::Examples examples_toIPv6 = {
+    {
+        "Usage example",
+        R"(
+WITH '2001:438:ffff::407d:1bc1' AS IPv6_string
+SELECT
+    hex(IPv6StringToNum(IPv6_string)),
+    hex(toIPv6(IPv6_string));
+        )",
+        R"(
+┌─hex(IPv6StringToNum(IPv6_string))─┬─hex(toIPv6(IPv6_string))─────────┐
+│ 20010438FFFF000000000000407D1BC1  │ 20010438FFFF000000000000407D1BC1 │
+└───────────────────────────────────┴──────────────────────────────────┘
+        )"
+    },
+    {
+        "IPv4-to-IPv6 mapping",
+        R"(
+SELECT toIPv6('127.0.0.1');
+        )",
+        R"(
+┌─toIPv6('127.0.0.1')─┐
+│ ::ffff:127.0.0.1    │
+└─────────────────────┘
+        )"
+    }
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_toIPv6 = {20, 1};
+    FunctionDocumentation::Category category_toIPv6 = FunctionDocumentation::Category::TypeConversion;
+    FunctionDocumentation documentation_toIPv6 = {description_toIPv6, syntax_toIPv6, arguments_toIPv6, returned_value_toIPv6, examples_toIPv6, introduced_in_toIPv6, category_toIPv6};
+
+    factory.registerFunction<detail::FunctionToIPv6>(documentation_toIPv6);
+
+    FunctionDocumentation::Description toString_description = R"(
```

Comment:
The markdown docs for toString say a lot about non-string conversions: https://clickhouse.com/docs/sql-reference/functions/type-conversion-functions#tostring

I think we should extract this information into the non-generated section of the markdown docs.